CMakeLists.txt

```diff
@@ -105,5 +105,10 @@ if (WITH_TESTING)
                 py_test(${src} SRCS ${src}.py ENVS ${PADDLE_PYTHON_PATH})
             endif()
         endforeach()
+        file(GLOB_RECURSE TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*_test_paddle.py")
```

Comment:
Can we just naming the unit test of paddle with   `*_test.py`?

Reply:
I think it is better to maintain a unified naming method with fluid and torch

Reply:
ok